js/customizer.js

```diff
@@ -0,0 +1,51 @@
+/* globals wp, UIEvent, jQuery, CodeMirror, _simpleCustomCSSCustomizerExports */
+/* exported simpleCustomCSSCustomizer */
+var simpleCustomCSSCustomizer = ( function( api, $ ) {
+	'use strict';
+
+	var self = {
+		editor: {},
+		textarea: {},
+		control: '',
+		element: ''
+	};
+
+	// Exports must be used to fill the previewData.
+	if ( 'undefined' !== typeof _simpleCustomCSSCustomizerExports ) {
+		$.extend( self, _simpleCustomCSSCustomizerExports );
+	}
+
+	self.init = function() {
+		$( document ).ready( function() {
+			var control;
+			control = api.control.instance( self.control );
+
+			// When our Customizer section is expanded, load CodeMirror.
+			api.section( control.section() ).container.on( 'expanded', function() {
+				self.loadCustomizer();
+			});
+		});
+	};
+
+	self.loadCustomizer = function() {
+		self.textarea = document.getElementById( self.element );
+		self.editor = CodeMirror.fromTextArea(
+			self.textarea,
+			{
+				lineNumbers: false,
+				lineWrapping: true
+			}
+		);
+		self.editor.on( 'change', function( cm ) {
+			var editorVal, changeEvent;
+			editorVal = cm.getValue();
+			self.textarea.value = editorVal;
+
+			changeEvent = new UIEvent( 'change' );
+			self.textarea.dispatchEvent( changeEvent );
+		});
```

Comment:
I'm not seeing two-way data binding here. I only see an edit to the code editor being pushed into the setting. But what happens when the underlying setting is changed? Will the editor get updated? Like you should be able to open your console and be able to do: `wp.customize( settingId ).set( 'body { color:red }' )` and this should result in the editor control also being updated.